src/test/java/org/inferred/freebuilder/processor/ProcessorTest.java

```diff
@@ -121,6 +121,95 @@ public void testPrefixlessInterface() {
         .runTest();
   }
 
+  @Test
+  public void testIgnoredEquality_allProperties() {
+    behaviorTester
+        .with(new Processor(features))
+        .with(SourceBuilder.forTesting()
+            .addLine("package com.example;")
+            .addLine("@%s", FreeBuilder.class)
+            .addLine("public interface DataType {")
+            .addLine("  @org.inferred.freebuilder.EqualsAndHashCode.Exclude int propertyA();")
```

Comment:
.addLine("  @%s int propertyA();", EqualsAndHashCode.Exclude.class)

This way IDE rename refactoring won't break the test :) Also below.

Reply:
:+1: